packages/core/src/node-utils/paths.ts

```diff
@@ -91,9 +91,6 @@ export const CustomTypesPaths = (cwd: string): Paths =>
 export const CustomPaths = (cwd: string): Paths => Paths(cwd, "");
 export const PackagePaths = (cwd: string): Paths => Paths(cwd, "node_modules");
 export const SMConfigPath = (cwd: string): string => path.join(cwd, "sm.json");
-export const LibrariesStatePath = (cwd: string): string =>
```

Comment:
I guess we should let user know that they can delete libraries-state

Reply:
I guess so but also there will be remaining mocks in .slicemachine, the goal would be more to tell them that they can no longer versioned `.slicemachine` all together on GitHub and clean that up. Only the mock config is a problem there but we’ll probably no longer support it.
But for sure we could have a changelog to tell them what they could clean. If no mock config, just remove `.slicemachine` all together would be the best recommandation

Reply:
cool, that'll need to go in the change-log when ever it's written :)